Userland/Libraries/LibJS/Tests/runtime-error-call-stack-size.js

```diff
@@ -7,15 +7,15 @@ test("infinite recursion", () => {
         infiniteRecursion();
     } catch (e) {
         expect(e).toBeInstanceOf(Error);
-        expect(e.name).toBe("Error");
-        expect(e.message).toBe("Call stack size limit exceeded");
+        expect(e.name).toBe("RangeError");
+        expect(e.message).toBe("Maximum call stack size exceeded");
     }
 
     expect(() => {
         JSON.stringify({}, () => ({ foo: "bar" }));
-    }).toThrowWithMessage(Error, "Call stack size limit exceeded");
+    }).toThrowWithMessage(RangeError, "Maximum call stack size exceeded");
 
     expect(() => {
         new Proxy({}, { get: (_, __, p) => p.foo }).foo;
-    }).toThrowWithMessage(Error, "Call stack size limit exceeded");
+    }).toThrowWithMessage(RangeError, "Maximum call stack size exceeded");
```

Comment:
Please revert all of this. The existing message is completely fine and a `RangeError` is a bad fit:

https://tc39.es/ecma262/#sec-native-error-types-used-in-this-standard-rangeerror

> Indicates a value that is not in the set or range of allowable values.

Our only source of truth is the spec, and this is non-standard anyway. If you're copying another engine's behaviour, you'll have to explain it - "Node does this" is not sufficient. This is not Node, and not trying to be it either.

Reply:
Fair enough. I was a bit iffy on that change, which is why I put it in a separate commit. That commit has been removed.